docs/configuration.md

```diff
@@ -3,4 +3,5 @@
 ### As a service:
 
 * I will take in environment variables natively to my service
-* I will not use configuration files where I can avoid it over using environment variables
+* Any environment specific configuration should be passed in as an environment variable
```

Comment:
I would say that private keys and certificates should always be injected in the environment, not as part of the code, kubernetes secrets are large enough for these